store/claims.js

```diff
@@ -0,0 +1,26 @@
+export const state = () => ({
+  claims: {}
+})
+
+export const getters = {
+  claims: state => {
+    return state.claims
+  }
+}
+
+export const mutations = {
+  SET_CLAIMS(state, claims) {
+    state.claims = claims
+  }
+}
+
+export const actions = {
+  async fetchClaims({ commit, state }, payload) {
+    console.log('fetching claims (admin)')
+    let response = await this.$adminArtifactClaimEndpoint.index(payload);
+    if (typeof response !== 'undefined') {
+      commit('SET_CLAIMS', response)
+      console.log('fetched claims (admin)')
+    }
+  }
+}
```

Comment:
Any reason this did not go in store/system.js?  We have an ADMIN_OFF mutation that clears stuff loaded into the store when the user logs off.  Rather than add another ADMIN_OFF in this file, and add it to the two we already invoke in the default.vue, maybe just move this into store/system.js?

Reply:
There's no specific reason as to why I kept the claim code separate. I just wanted to make it seem a little cleaner. I wasn't aware of the ADMIN_OFF mutation we had. I can just move this into store/system.js itself.

Reply:
Thanks, I think that will be easiest for now.  I prefer your approach to split things out into separate files, but when handling admin transitions in the UI, it's always best if we can keep the ADMIN_OFF state mutator as a single call that handles everything about the admin transition, for longer term maintenance.  So we would want to define some kind of hook system for ADMIN_OFF, where state "submodules" could register a lambda to do the right thing on mutation... and quite likely vue provides something for us already.

Reply:
That makes sense David. Maybe we could try exploring the "hook system" as an enhancement in a future PR.